lib/cmds/fhir_cmds/list.js

```diff
@@ -8,13 +8,13 @@ const url = require('url');
 const jmespath = require('jmespath');
 const fs = require('fs');
 
-exports.command = 'list <type>';
-exports.desc = 'List FHIR resources by type <type>.';
+exports.command = 'list <type> <project>';
+exports.desc = 'List FHIR resources by type <type> <project>.';
 exports.builder = yargs => {
   yargs.positional('type', {
     describe: 'The FHIR resource type.',
     type: 'string'
-  }).option('project', {
+  }).positional('project', {
```

Comment:
Do we have other cases where optional-looking params are required?  If so, we should keep this as a param because if anyone has documentation or saved scripts that use this command would break if they were doing it right from the beginning (`lo fhir list Procedure --project=<pid>`).  Either way, this is a breaking change technically.. but to just make the `--project` now required might be a more graceful approach.  If, OTOH, that's a can of worms, I could get behind this change for its simplicity.

Reply:
No examples of an optional param that is required, but that is a good point.

I did a quick scan of GH and every repo where this is used has the `--projectId` param, so I'd rather not break for what's in version control.  I'll go ahead an update to keep it as an option